packages/kotti-ui/source/kotti-inline-edit-2/KtInlineEdit2.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+	<div :class="objectClass">
+		<label class="form-label" v-text="label" />
+		<div v-if="isEditing">
+			<component
+				:is="inputComponent"
+				v-bind="$attrs"
+				class="form-input"
+				type="text"
+				:value="currentValue"
+				@change="handleChange"
+				@input="handleInput"
+				v-text="currentValue"
+			/>
```

Comment:
I'm highly in disfavor of the component,

I think it's much easier to explicitly have `KtFieldText` and `KtFieldTextArea` depending on `isMultiLine`

but the only obstacle would be the `@change`

Reply:
why does `@change` is an obstacle for having `KtFieldText` and `KtFieldTextArea` ?

Reply:
becuase currently we don't emit the `blur` event from those fields

so we need to change that